packages/@aws-cdk/aws-scheduler-alpha/lib/schedule.ts

```diff
@@ -30,6 +33,54 @@ export interface ISchedule extends IResource {
   readonly key?: kms.IKey;
 }
 
+export interface ScheduleTargetProps {
+  /**
+   * The text, or well-formed JSON, passed to the target.
+   *
+   * If you are configuring a templated Lambda, AWS Step Functions, or Amazon EventBridge target,
+   * the input must be a well-formed JSON. For all other target types, a JSON is not required.
+   *
+   * @default - The target's input is used.
+   */
+  readonly input?: ScheduleTargetInput;
+  /**
+   * The maximum amount of time, in seconds, to continue to make retry attempts.
+   *
+   * @default - The target's maximumEventAgeInSeconds is used.
+   */
+  readonly maximumEventAge?: Duration;
+  /**
+   * The maximum number of retry attempts to make before the request fails.
+   *
+   * @default - The target's maximumRetryAttempts is used.
+   */
+  readonly maximumRetryAttempts?: number;
+}
+
+export interface ScheduleTargetProps {
+  /**
+   * The text, or well-formed JSON, passed to the target.
+   *
+   * If you are configuring a templated Lambda, AWS Step Functions, or Amazon EventBridge target,
+   * the input must be a well-formed JSON. For all other target types, a JSON is not required.
+   *
+   * @default - The target's input is used.
+   */
+  readonly input?: ScheduleTargetInput;
+  /**
+   * The maximum amount of time, in seconds, to continue to make retry attempts.
+   *
+   * @default - The target's maximumEventAgeInSeconds is used.
+   */
+  readonly maximumEventAge?: Duration;
+  /**
+   * The maximum number of retry attempts to make before the request fails.
+   *
+   * @default - The target's maximumRetryAttempts is used.
+   */
+  readonly maximumRetryAttempts?: number;
```

Comment:
```suggestion
  readonly maxRetryAttempts?: number;
```

Reply:
Renamed to `retryAttempts` as in [target](https://github.com/aws/aws-cdk/blob/fc3be313daf40bbf12b55f0b3ed3027505ca3a87/packages/%40aws-cdk/aws-scheduler-targets-alpha/lib/target.ts#L62) class.